examples/prompting/Basic_Reasoning.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
<p>Gemini is the consumer chat app. Make sure you only talk about "Gemini" when you mean the app (this applies everywhere in every guide).</p><p> </p><p>For Gemini API examples, you can say "The Gemini API" or just "the models" is often enough.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/150/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>